src/api/informer.rs

```diff
@@ -32,8 +24,9 @@ type WatchQueue<K> = VecDeque<WatchEvent<K>>;
 /// It caches WatchEvent<K> internally in a queue when polling.
 /// A user should drain this queue periodically.
 #[derive(Clone)]
-pub struct Informer<K> where
-    K: Clone + DeserializeOwned + KubeObject
+pub struct Informer<K>
+where
+    K: Clone + DeserializeOwned + KubeObject,
 {
     events: Arc<RwLock<WatchQueue<K>>>,
```

Comment:
Now that we're having poll return a `Stream<Item = Result<T>>` this events member is no longer used. Do you want me to remove it? I can replace it with `PhantomData<T>` to keep the type information tied to the `Informer` rather than the `poll` function itself.

Reply:
Yeah, might as well. I think we might need to rename some methods to indicate the new streaming interface as well. But not sure what a good name is yet.